grafana-cloud-metrics.yaml

```diff
@@ -20,7 +19,7 @@
 heat_template_version: 2013-05-23
```

Comment:
no that's the heat template version we're using.  Apparently they versioned template syntax by date.